credits.html

```diff
@@ -0,0 +1,274 @@
+                                <li><a href="http://devdocs.astropy.org/" target="_blank">In Development</a></li>
+                                <li><a href="https://astropy.readthedocs.io/en/lts/index.html" target="_blank">Long-Term Support</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </li>
+				<li><a href="affiliated/index.html">Affiliated Packages</a></li>
+                <li><a href="team.html">Team</a></li>
+                <li><a href="credits.html">Credits</a></li>
+			</ul>
+		</div>
+		<div class="search pull-right">
+			<form action="http://docs.astropy.org/en/stable/search.html" method="get">
+			  <input type="text" name="q" placeholder="Search Documentation" />
+			  <input type="hidden" name="check_keywords" value="yes" />
+			  <input type="hidden" name="area" value="default" />
+			</form>
+		</div>
+	</nav>
+
+    <section>
+    <h1 id="astropy-credits">Astropy Credits<a class="paralink"
+    href="#astropy-credits" title="Permalink to this headline">¶</a></h1>
+
+    <p>The Astropy Project is made possible through the hard work of hundreds of
+    people in the community. Contributions take many forms, from participating
+    in online forums, being an Astropy sub-committee member, giving talks,
+    writing tutorials and documentation, writing code, making releases,
+    organizing conferences, and much much more.</p>
+
+    <p>In this page we specially recognize the people and organizations that
+    have made significant contributions to the Astropy project. This comes in
+    the form of <a href="#lifetime-credits">lifetime contributors</a> and <a
+    href="#institutional-support">institutional support</a> that provides both
+    direct and indirect funding for Astropy.
+
+    </p>
+
+
+</section>
+
+<section>
+<h1 id="lifetime-credits">Lifetime Contributors<a class="paralink"
+href="#lifetime-credits" title="Permalink to this headline">¶</a></h1>
+
+    <p>Here we list individuals who estimate they have spent 2000 hours or more,
+    equivalent to full time for a year, working on and contributing to the
+    Astropy Project. These individuals have demonstrated a long-term commitment
+    to the project in many areas and exemplify the high standards we seek to
+    achieve.
+    </p>
+
+    <h3>Adrian Price-Whelan</h3>
+    Adrian is the lead coordinator and developer behind the Learn Astropy and
+    Astropy tutorials initiatives. He was an initial contributor of the concept
+    and code for the Quantity class in astropy.units and serves as lead
+    developer of astropy.coordinates. In the latter role he played a key role in
+    adding support for velocity data such as proper motions in
+    astropy.coordinates.
+
+    <h3>Brigitta Sipőcz</h3>
+    Since coming to the project as a Google Summer of Code (GSoC) student in
+    2014, Brigitta has been an active and valued member of the Astropy community
```

Comment:
```suggestion
    Since coming to the project in
    2014, Brigitta has been an active and valued member of the Astropy community
```
We don't say that E.g. Eric joined the project as a student, either.

Reply:
But thinking about it a little more, we might want to keep it in here and then add "s a student" to Eric, too, and say 'as a post-Doc' for Tom R and me. Not sure about the others, but it might be a good motivation for future, early career (e.g. student, Post-Doc) contributors!